### PR TITLE
Remove hardcoded targetframeworkversion

### PR DESCRIPTION
**Pull request by Mihaiii**
`master` branch is for SC 2.3, I assume.

After this PR is merged, I will make the change for `develop` branch (I assume it's the official branch for SC 2.4). 

A release needs to be made first before changing `level1`. 
For clarity, I don't intend to make the release, unless someone specifies it.

For https://github.com/Starcounter/Home/issues/354

### CODE REVIEW

File: src/Starcounter.VS.CSApplicationProjectTemplate/ApplicationProjectTemplate.csproj [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>$safeprojectname$</RootNamespace>
     <AssemblyName>$safeprojectname$</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v$targetframeworkversion$</TargetFrameworkVersion>
```
Comment (miyconst) [inline src/Starcounter.VS.CSApplicationProjectTemplate/ApplicationProjectTemplate.csproj:14]: @Mihaiii was this tested? Are you sure that `v` is not included into the `$targetframeworkversion$` variable value?

Comment (Mihaiii) [reply src/Starcounter.VS.CSApplicationProjectTemplate/ApplicationProjectTemplate.csproj:14]: Yes, it was tested with and without the "v". "v" is not included into the `$targetframeworkversion$` variable value.

Comment (miyconst) [reply src/Starcounter.VS.CSApplicationProjectTemplate/ApplicationProjectTemplate.csproj:14]: Okay, thanks for the clarification. I also add Urban to review, but it's approved from me.

**Review by miyconst — APPROVED** [review]
Please wait for Urban's review as well.

**Review by ubbeK — APPROVED** [review]
looks good.

Out of curiosity, did you any documentation regarding `targetframeworkversion` variable? Did some googling but didn't find much regarding it when creating a new project.

**Comment by Mihaiii** [comment]
> Out of curiosity, did you any documentation regarding targetframeworkversion variable? Did some googling but didn't find much regarding it when creating a new project.

No, I haven't.
Where should it be mentioned considering that it's a bug fix, not a feature?

**Comment by Mihaiii** [comment]
@ubbeK , I'm going to make the merge because apparently github's interface doesn't allow to make two PRs from the same branch.

Please let me know here or on Slack if I need to update anything else.

**Pull request merged**